lib/hostel/site.rb

```diff
@@ -15,7 +15,9 @@ def domain
     end
     
     # Constructs a url for a path on this site
-    def build_path(path, secure: true, path_args: {})
+    # Note:  request_host is ignored now, but it is here for backward
+    # compatibility with old callers.
```

Comment:
I'm a little confused by what this means. Maybe if you link me to an example of one of these "old callers" I might understand better, but why can't we just update the function calls to not pass a parameter that we aren't going to use?